src/stylesheets/_typography.scss

```diff
@@ -66,28 +66,116 @@
   @include responsiveness.font($responsive: 2.3vw, $min: 12px, $max: 18px);
 }
 
+@mixin admin-card {
+  padding: 20px;
+  background: rgba(242, 250, 255, 0.21);
+  box-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+  border-radius: 10px;
+  margin-bottom: 30px;
+}
+
+
 @mixin form-prompts {
   color: var(--color-black);
   font-family: "Lato", sans-serif;
   font-style: normal;
-  font-weight: lighter;
+  font-weight: 400;
 
   @include responsiveness.font($responsive: 2.3vw, $min: 18px, $max: 28px);
 }
 
-@mixin form-titles {
+@mixin form-component-width-large {
```

Comment:
This is probably going to be a story on it's own fixing the responsiveness on multiple screen sizes since that's still far from complete but I'll definitely add this!